loadtest/control/simulcontroller/actions.go

```diff
@@ -876,6 +876,22 @@ func createMessage(u user.User, channel *model.Channel, isReply bool) (string, e
 		message = control.AddLink(message)
 	}
 
+	// 10% of messages will contain a permalink
+	if rand.Float64() < 0.10 {
+		post, err := u.Store().RandomPostForChannel(channel.Id)
+		if err != nil && !errors.Is(err, memstore.ErrPostNotFound) {
+			return "", err
+		}
```

Comment:
Okay with not returning if the post is not found but we can't go ahead with the rest of this case or it would panic.

Reply:
:facepalm: